src/app/core/directives/permission.directive.ts

```diff
@@ -0,0 +1,27 @@
+import {Directive, Input, OnInit, TemplateRef, ViewContainerRef} from '@angular/core';
+import {AuthService} from "../../auth/services/auth.service";
+
+@Directive({
+  selector: '[appPermission]'
+})
+export class PermissionDirective implements OnInit {
+
+  @Input('appPermission')
+  userConnected!: boolean;
```

Comment:
J'ai l'impression que tu ne l'utilise pas ce booléen. Je pense qu'on peut le supprimer.

Reply:
Oui, t'as raison, j'ai merdé quelques part. C'est pas bon du tout ça.
Mais je ne peux pas l'enlever, sinon j'ai un message d'erreur : 
*appPermission souligné en rouge : Property appPermission is not provided by any applicable directive on an embedded template